modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -7,39 +7,39 @@ Create AS2 identifiers for your organization as host and for your trading partne
 [%header%autowidth.spread]
 |===
 | Setting | Description | Required
-| Endpoint Name
-| The automatically generated name used to identify the endpoint, which you can subsequently modify manually.
-| Yes
+|Owner
+|The host or trading partner that owns the endpoint.
+|Yes
 
-| Owner
-| The host or trading partner that owns the endpoint.
-| Yes
+|Usage
+|The usage of the endpoint. For receive endpoints, the value in this field is `Receive from partners`. The value in this field is automatically generated based on the type of endpoint created.
+|No
 
-| Protocol
-| The protocol previously selected in the *Protocol* field of the *New endpoint: Receive from partners* window. This value cannot be subsequently modified.
-| Yes
+|Protocol
+|The protocol previously selected in the *Protocol* field of the *New endpoint: Receive from partners* window. This value cannot be subsequently modified.
+|Yes
 
-| Description
-| The user-supplied description of the purpose and use of the endpoint.
-| No
+|Endpoint Name
```

Comment:
See Line 10: Is the UI inconsistent in its capitalization (Keystore name, Endpoint Name)?
(what a surprise....)
We do have to quote the UI verbatim, when quoting. We can fix when writing about.

Reply:
It is lower case - thanks for catching it!